.golangci.yml

```
@@ -1,20 +1,34 @@
+formatters:
+  enable:
+    - gci
+    - gofmt
+    - gofumpt
+    - goimports
+  exclusions:
+    generated: lax
+  settings:
+    gci:
+      sections:
+        - standard
+        - default
+        - prefix(github.com/shirou)
```

There is a problem for me here

gci supersedes goimports
gofumpt supersedes gofumpt

So here, your conf should be to something as simple as this

```suggestion
formatters:
  enable:
    - gci
    - gofumpt
  exclusions:
    generated: lax
  settings:
    gci:
      sections:
        - standard
        - default
        - prefix(github.com/shirou)
```

The only reasons I would like to see gofmt with gofumpt is if you use extra features such as [simplify](https://pkg.go.dev/cmd/gofmt#hdr-The_simplify_command) and [rewrites-rules](https://pkg.go.dev/cmd/gofmt)

```suggestion
formatters:
  enable:
    - gci
    - gofmt
    - gofumpt
  exclusions:
    generated: lax
  settings:
    gofmt:
      simplify: true
      rewrite-rules:
        - pattern: 'interface{}'
          replacement: 'any'
    gci:
      sections:
        - standard
        - default
        - prefix(github.com/shirou)
```